### DISCUSSION TITLE
download

### DISCUSSION THREAD

**Discussion by bahabryra** [copy/v86#1152] [General]
is it possible for someone to download this and run it on a html file including the vm if so how??? and could you add it to your tags if possible???

EDIT: IF I'm The one who made this then why are you locking me from commenting on this??? 
EDIT 2:And Why Am I blocked from Making issues

**Comment by SuperMaxusa** [comment]
If you can run a local HTTP file server (eg. `python3 -m http.server` or `caddy file-server`) - no problems, if you run from a local folder - could be some CORS errors in browsers, see https://github.com/copy/v86/issues/1029.
